src/cascade/input_data/configuration/builder.py

```diff
@@ -256,16 +262,24 @@ def covariate_to_measurements_nearest_favoring_same_year(measurements, covariate
     """
     # Rescaling the age means that the nearest age within the year
     # will always be closer than the nearest time across a full year.
-    tree = spatial.KDTree(list(zip(
-        covariates[["age_lower", "age_upper"]].mean(axis=1) / 240,
-        covariates[["time_lower", "time_upper"]].mean(axis=1),
-        covariates["x_sex"]
-    )))
-    _, indices = tree.query(list(zip(
-        measurements[["age_lower", "age_upper"]].mean(axis=1) / 240,
-        measurements[["time_lower", "time_upper"]].mean(axis=1),
-        measurements["x_sex"],
-    )))
+    tree = spatial.KDTree(
```

Comment:
This area might conflict with Natalie's work. Looks like a reformatting, not a change in what it does.